### DISCUSSION TITLE
How do we configure gRPC health check for VM workloads?

### DISCUSSION THREAD

**Discussion by CadenGuo** [istio/istio#53304] [General]
As of now, we can only configure 3 types of readiness probes for VM workloads:

1. httpGet
2. tcpSocket
3. exec

grpc prober is not available yet. Can we somehow configure it for the VM workloads? Or do we plan to support it?

**Comment by howardjohn** [comment]
Currently its not implemented. I am not opposed to adding it if someone wants to do the work
